interfaces/effect_player.hpp

```diff
@@ -9,6 +9,11 @@ namespace bnb::oep::interfaces
     class effect_player;
 }
 
+namespace bnb::interfaces
+{
+    class js_callback;
```

Comment:
this is bad. No need to add class from Banuba SDK here.

OEP-module does not depend on Banuba SDK